src/lib/import-export/import/events.ts

```diff
@@ -0,0 +1,32 @@
+export const HandlerEvents = {
+	BACKUP_HANDLER_START: 'handler_start',
+	BACKUP_HANDLER_PROGRESS: 'handler_progress',
+	BACKUP_HANDLER_COMPLETE: 'handler_complete',
+	BACKUP_HANDLER_ERROR: 'handler_error',
```

Comment:
I'm thinking to change these events to specify that they are related to extract the backup. E.g.:
```
BACKUP_EXTRACT_START: 'backup_extract_start',
BACKUP_EXTRACT_PROGRESS: 'backup_extract_progress',
BACKUP_EXTRACT_COMPLETE: 'backup_extract_complete',
BACKUP_EXTRACT_ERROR: 'backup_extract_error',
```

Reply:
It makes sense. It would make it consistent with other event names.

Reply:
I created a PR to update this: https://github.com/Automattic/studio/pull/384